external/test-runtime/XUnit.Runtime.depproj

```diff
@@ -86,6 +86,16 @@
     <PackageToInclude Include="$(XUnitRunnerPackageId)" />
   </ItemGroup>
 
+  <PropertyGroup Condition="'$(TargetGroup)' == 'uap'">
+    <UAPToolsPackageVersion>1.0.5</UAPToolsPackageVersion>
```

Comment:
I was mainly asking because I was wondering if it should be in dependencies.props. For now given that we will only ever manually update it and we use it only here I think it is OK to keep it this way.